util/json_commands.lua

```diff
@@ -290,6 +305,26 @@ local function run_shell_command(request)
 end
 
 
+local function run_psql_command(command)
+
+  local pg = pgmoon.new({
```

Comment:
This creates a new pg connection on each command, right? I'm afraid this can lead to running out of descriptors if a test file is long. If caching the connection would add more trouble than it's worth, then perhaps closing the connection explicitly would be good enough?

Reply:
Did you notice any speed changes btw? (I'd think opening connections would be slower than closing them)

Reply:
I did not notice any differences. Most of the time is still spent installing kong versions.